src/transformers/pipelines/base.py

```diff
@@ -1072,6 +1072,10 @@ def __call__(self, inputs, *args, num_workers=None, batch_size=None, **kwargs):
             )
         elif is_iterable:
             return self.iterate(inputs, preprocess_params, forward_params, postprocess_params)
+        elif isinstance(self, ChunkPipeline):
```

Comment:
I'm not a huge fan of this condition.

But it is explicit and quite a minimal code change, that's why I went for it. If anyone has better suggestions I'm taking them